### DISCUSSION TITLE
php 8

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#14072] [How to]
Today I have installed YetiForceCRM 6.1 on an openSUSE 15.1 test server running apache 2.4 and **php 8.0.1**. Although I have not tested the system i detail yet, I have found no issued with using **php 8.0.1**.

**Comment by mariuszkrzaczkowski** [comment]
In 6.1. version a lot of libraries aren't compatible with PHP 8, we changed the requirements since 6.2 and abandoned PHP 7.2 and as a result most libraries are compatible with PHP 8 while PHP 7.2 blocked them.
Recently, we have seen a lot of progress with the changes for PHP 8 and already almost all libraries support the new version.
Soon, we will run GitDev with PHP 8 for better compatibility with the new version.
In 6.1 version, try to run the installation of libraries on PHP 8 and you will see that it won't work.
